# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: d82b6476a18d
# Full Hash: d82b6476a18d1dedfe0cb5be42ee9c7040fcf4da
# Author: Dan Glastonbury <dan.glastonbury@gmail.com>
# Date: 2020-10-21 09:51:28
# Regressor Bug: 1595994
# File Overlap Count: 1
# Description:
#   Bug 1595994 - P10: Allow FFMPEG decoding in RDD process. r=mattwoodrow
#   
#   Disabled by default.
#   
#   Depends on D54884
# ==============================================================================

diff -r 9914d2e0f6ca -r d82b6476a18d modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Tue Oct 20 23:27:13 2020 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Tue Oct 20 23:27:21 2020 +0000
@@ -7069,6 +7069,13 @@
   value: 5000
   mirror: always
 
+#ifdef MOZ_FFMPEG
+- name: media.rdd-ffmpeg.enabled
+  type: RelaxedAtomicBool
+  value: false
+  mirror: always
+#endif
+
 #ifdef MOZ_FFVPX
 - name: media.rdd-ffvpx.enabled
   type: RelaxedAtomicBool
